lib/src/cupertino/cupertino_toolbar.dart

```diff
@@ -25,6 +25,8 @@ class CupertinoPopoverToolbar extends StatefulWidget {
     this.borderRadius = 12.0,
     this.padding,
     this.backgroundColor = const Color(0xFF333333),
+    this.elevation = 0.0,
```

Comment:
Should there be asserts on these constructors, too?

Reply:
Added.